solutions/usaco-712.mdx

```diff
@@ -108,3 +108,44 @@ int main() {
 </CPPSection>
 
 </LanguageSection>
+
+## Alternative Solution
```

Comment:
since there's already an alternative solution above, can you rename the solutions to solution 1, solution 2, solution 3?

Reply:
yeah that sounds better